internal/cmdrpkgclone/command.go

```diff
@@ -96,8 +100,8 @@ func newRunner(ctx context.Context, rcg *genericclioptions.ConfigFlags) *runner
 
 	c.Flags().StringVar(&r.strategy, "strategy", string(porchapi.ResourceMerge),
 		"update strategy that should be used when updating this package; one of: "+strings.Join(strategies, ","))
-	c.Flags().StringVar(&r.directory, "directory", "/", "Directory within the repository where the upstream package is located.")
-	c.Flags().StringVar(&r.ref, "ref", "main", "Branch in the repository where the upstream package is located.")
+	c.Flags().StringVar(&r.directory, "directory", "", "Directory within the repository where the upstream package is located.")
+	c.Flags().StringVar(&r.ref, "ref", "", "Branch in the repository where the upstream package is located.")
```

Comment:
I removed these defaults because otherwise it is impossible to tell whether the user set the flag explicitly or not. Since we are trying to infer the ref from the URL, we need to be able to throw an error if the user sets the flag and also includes the ref in the URL and they contradict each other. I set the default manually below in L182-L187 after this error check occurs.